assets/pages/bundles-landing/components/stackedBundle.jsx

```diff
@@ -277,23 +277,12 @@ function getDigitalAttrs(subsLinks: SubsUrls): SubscribeAttrs {
   return Object.assign({}, bundles.digital, { ctaLink: subsLinks.digital });
 }
 
-function WhyContribute(props: {shouldEncourageMonthly: boolean}) {
-  if (props.shouldEncourageMonthly) {
-    return (
-      <p className="bundles__why-contribute">
-        Your contribution funds and supports the&nbsp;Guardian&#39;s journalism.
-        If you’re able, please consider
-        <strong> monthly</strong> support – it will help to fund our journalism for the long term.
-      </p>
-    );
-  }
+const WhyContribute = () => (
+  <p className="bundles__why-contribute">
+    Your contribution funds and supports the&nbsp;Guardian&#39;s journalism.
```

Comment:
@Amohkhan should this be 'The Guardian' instead of 'the Guardian'?

Reply:
Good spotted, the fix is in: 4f5577c0b221410738a7927fea0803c3a212eaa8